### PR TITLE
adds line numbers and cleanup proc

### PR DESCRIPTION
**Pull request by freesig**
This makes #[autotrace] automatically add the file and line number.
I also cleaned up the newrelic macro so that it only adds #block once. Other wise our code size can grow a lot.
There's an issue with having both macros on the same function, it wipes out the line information.

### CODE REVIEW

**Comment by freesig** [comment]
Whoops need to update the tests

File: crates/tracing_macros/src/newrelic_trace.rs [hunk 1]
```diff
@@ -22,6 +22,30 @@ impl NewRelicTrace {
...
+                {
+                    None
+                }
+            }
+            else
+            {
+                None
+            };
+            #block
+            /*
```
Comment (maackle) [inline crates/tracing_macros/src/newrelic_trace.rs]: hmm why does this appear to be all additions? I thought this was in already?

**Review by maackle — APPROVED** [review]
Looks good after test update

**Review by StaticallyTypedAnxiety — APPROVED** [review]
Awesome!

File: crates/tracing_macros/src/newrelic_trace.rs (hunk 1 above)
Comment (freesig) [reply crates/tracing_macros/src/newrelic_trace.rs]: Because it was #block where None was if that makes sense

**Pull request merged**